extras/rc.nginx/nginx-log/env/LOG_SCRIPT

```diff
@@ -0,0 +1,2 @@
+n4 s500000 $LOGDIR_PATH
```

Comment:
After testing this PR on my machine with default settings, it seems `nginx` doesn't add time stamps to logs printed on `stdout`. Here is an example output of `/var/log/nginx/current`:

```
nginx: [emerg] bind() to 0.0.0.0:22 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:22 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:22 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:22 failed (98: Address already in use)
nginx: [emerg] bind() to 0.0.0.0:22 failed (98: Address already in use)
nginx: [emerg] still could not bind()
```

That means, we have to add the `T` directive to our `LOG_SCRIPT`:

```suggestion
n4 s500000 T $LOGDIR_PATH
```

Reply:
Right, haven't noticed that, thanks.